app/views/registrations/edit_v2.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, t('registrations.edit_registration.title', person: @user.person.name, comp: @competition.name)) %>
+<% add_to_packs("auto_numeric") %>
```

Comment:
I don't think you need this pack here, because you're only rendering a React component and that component uses imports through webpack.

The `add_to_packs` hack is only necessary if you want to access Webpack code in Sprockets